src/uwtools/tests/drivers/test_jedi.py

```diff
@@ -98,35 +98,6 @@ def test_JEDI_driver_name(driverobj):
     assert driverobj.driver_name() == JEDI.driver_name() == "jedi"
 
 
-def test_JEDI_files_copied(driverobj):
```

Comment:
This seemed to be the only pre-existing tests of these tasks that failed given they were patching objects in this way. I left all the other passing ones in place. Does it make sense to replace these two in JEDI with tests similar to the other drivers or will the test of the FileStager methods suffice?